deps.edn

```diff
@@ -454,23 +457,121 @@
+    com.github.seancorfield/depstar   {:mvn/version "2.1.303"}
+    expound/expound                   {:mvn/version "0.7.0"}                    ; better output of spec validation errors
+    io.github.borkdude/grasp          {:mvn/version "0.0.3"}
+    io.github.clojure/tools.build     {:git/tag "v0.9.3" :git/sha "e537cd1"}
+    org.clojure/tools.deps.alpha      {:mvn/version "0.12.985"}
+    org.fedorahosted.tennera/jgettext {:mvn/version "0.15.1"}}
+
+   :jvm-opts
+   ["-Dclojure.main.report=stderr"
+    "-XX:-OmitStackTraceInFastThrow"
+    "-XX:+ShowCodeDetailsInExceptionMessages"]}
+
+  ;; Build everything:
+  ;;
+  ;;    clojure -X:drivers:build:build/all
+  ;;    clojure -X:drivers:build:build/all :edition :ee
+  ;;
+  ;; Run just a specific build step:
+  ;;
+  ;;    clojure -X:drivers:build:build/all :steps '[:version]'
+  ;;
+  ;; the various steps available are:
+  ;;
+  ;;    :version :translations :frontend :licenses :drivers :uberjar
+  :build/all
+  {:exec-fn build/build-cli}
+
+  ;; build just the uberjar (without i18n, drivers, etc.)
+  ;;
+  ;; clojure -X:build:build/uberjar
+  ;; clojure -X:build:build/uberjar :edition :ee
+  :build/uberjar
+  {:exec-fn build.uberjar/uberjar}
+
+  ;; List dependencies without a license. Not really 100% sure why this needs `:dev`
+  ;;
+  ;; clojure -X:build:build/list-without-license
+  :build/list-without-license
+  {:exec-fn build/list-without-license}
+
+  ;; Build a single driver.
+  ;;
+  ;;   clojure -X:build:drivers:build/driver :driver :sqlserver :edition :oss
+  :build/driver
+  {:exec-fn build-driver/build-driver}
+
+  ;; Build all of the drivers.
+  ;;
+  ;;    clojure -X:build:drivers:build/drivers
+  ;;    clojure -X:build:drivers:build/drivers :edition :ee
+  :build/drivers
+  {:exec-fn build-drivers/build-drivers}
+
+  ;; Verify that a driver JAR looks correct.
+  ;;
+  ;;    clojure -X:build:build/verify-driver :driver :mongo
+  :build/verify-driver
+  {:exec-fn verify-driver/verify-driver}
+
+  ;; Build i18n artifacts.
+  ;;
+  ;;    clojure -X:build:build/i18n
+  :build/i18n
+  {:exec-fn i18n.create-artifacts/create-all-artifacts!}
+
+  ;; Build everything, and run the release steps.
+  ;;
+  ;;    clojure -X:drivers:build:build/release
+  ;;    clojure -X:drivers:build:build/release :steps '[:build-uberjar :upload-uberjar]'
+  ;;
+  ;; Available steps:
+  ;;
+  ;;    :build-uberjar
+  ;;    :upload-uberjar
+  ;;    :push-git-tags
+  ;;    :publish-draft-release
+  ;;    :publish-elastic-beanstalk-artifacts
+  ;;    :update-version-info
+  :build/release
+  {:exec-fn release/release}
+
+  ;; extra paths and deps for working on build scripts, or running the tests.
+  :build-dev
+  {:extra-paths ["bin/build/test"]
+
+   ;; some of these are here just for to have jars on the classpath for testing the licenses
+   ;; stuff (see [[build.licenses-test]])
+   :extra-deps
```

Comment:
All of these were chosen as examples from our current deps. They aren't needed to be added.

Reply:
fixed